includes/orders/functions/actions.php

```diff
@@ -297,6 +315,25 @@ function edd_add_manual_order( $args = array() ) {
 					}
 				}
 
+				// Maybe store order tax.
+				if ( $tax_rate ) {
+					$description = $tax_rate->name;
+					if ( ! empty( $tax_rate->description ) ) {
+						$description = $tax_rate->description;
+					}
+					// Always store tax rate, even if empty.
```

Comment:
@robincornett I'm not sure what would be empty here/what is being forced here?

Reply:
@spencerfinnell  The `$tax_rate->name` is always set to the country code (eg `US`). The description may be blank. If the tax rate is regional, the `$tax_rate->description` will be the region code (eg `TN`). So this is setting the adjustment description to the name initially, and then overriding it with the more specific value if it exists.

Reply:
@robincornett That makes sense. Perhaps a comment above the `! empty()` check explaining that would be clearer. I thought the comment might have been referring to the result of `edd_get_tax_rate_by_location()`.